docs/flow-results-connector.md

```diff
@@ -0,0 +1,112 @@
+## FLOW RESULTS PACKAGE CONNECTOR
+
+This connector is designed in Apache NiFi to automate the process of transferring flow-results packaged data to a table in a Postgres database or store AVRO formatted response results on disk. The connector ingests data in three different ways:
+
+1. __Flow results API endpoint__
+The schema for the table is derived from the descriptor of a package retrieved from `/flow-results/packages/[form_uuid]`. Once the schema for the table is generated, a table is created in the database. Data is then retrieved from the response endpoint: `/flow-results/packages/[form_uuid]/responses`. The connector requires a JSON configuration with the keys `form_uuid`, `base_url` and optionally, the `table_name` the user desires the data to be stored in. The JSON config can is formatted as below:
```

Comment:
I think the same applies to the API as well. It would be,
```JSON
{
  "id": "[Data package id in UUID format]",
  "table_name": "[preferred Postgres table name]",
  "base_url": "[Base URL to get data from]"	
}
```